src/server/player.js

```diff
@@ -9,6 +9,7 @@ export class Player {
     this.game = null;
     this.name = '';
     this.errorMessage = null;
+    this.score = 0;
```

Comment:
Think this should be stored against the game

Reply:
removing this for now as I'll do the scoring in another PR. But think that makes sense